python/pyspark/sql/column.py

```diff
@@ -462,9 +488,39 @@ def isin(self, *cols):
     >>> df.select(df.name).orderBy(df.name.desc()).collect()
     [Row(name=u'Tom'), Row(name=u'Alice')]
     """
+    _desc_nulls_first_doc = """
+    Returns a sort expression based on the descending order of the given column name and null values
+    return before non-null values
+
+    >>> from pyspark.sql import Row
+    >>> df = spark.createDataFrame([
+    ...     Row(name=u'Tom', height=80),
+    ...     Row(name=None, height=None),
+    ...     Row(name=u'Alice', height=None)
+    ... ])
```

Comment:
I think, things like `spark.createDataFrame([('Alice', 1)], ['name', 'height'])` would be simpler.